### PR TITLE
fix(DB/Quest): Strengthen the Ancients

### PR DESCRIPTION
**Pull request by mpfans**
## Changes Proposed:
-  fix Woodlands Walker gossip

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/12123

## SOURCE:

https://www.youtube.com/watch?v=XH4slsQWlPw

## Tests Performed:

- tested ingame
- 

## How to Test the Changes:

1.  .q a 12092 (horde 12096)
2.talk Woodlands Walker (npc id 26421)

## Known Issues and TODO List:

- There is no gossip option for this NPC. After accepting SQL, the server will start with an error

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by tkn963** [comment]
I imagine you should be specific with the npcflag. If a user, for whatever reason, happen to run the query more than once it'll add 1 to the npcflag every time. Especially if a user is told to remove the entries from the updates table due to various issues.

**Comment by acidmanifesto** [comment]
just a update. I have dropped a request in a few dev channels in seeing about correcting the dry run error. we will have to wait until a solution is provided.

**Comment by Nyeriah** [comment]
The sql is missing a semicolumn as the CI says

File: data/sql/updates/pending_db_world/rev_1655718205934882580.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- fix Woodlands Walker gossip
+UPDATE `creature_template` SET `npcflag`=`npcflag` + 1 WHERE `entry` = 26421
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1655718205934882580.sql:2]:
```suggestion
UPDATE `creature_template` SET `npcflag`=`npcflag` |1 WHERE `entry` = 26421;
```

**Comment by acidmanifesto** [comment]
@mpfans a pr was merged that should make your pr pass. can you please fetch the upstream of your PR so the test can re-run?

**Comment by mpfans** [comment]
completed

**Review by acidmanifesto — APPROVED** [review]

**Comment by pangolp** [comment]
@mpfans so that the validation does not throw an error, try to change the script for this, where you add an extra_flag, created for this purpose, to ignore those npc, which have the flag of gossip, but that are not in the gossip flag, but that are not in the gossip flag.

```sql
UPDATE `creature_template` SET `npcflag`=`npcflag`|1, `flags_extra`=`flags_extra`|16777216 WHERE `entry`=26421;
```

**Pull request merged**